Flow.Launcher.Plugin/Interfaces/IPublicAPI.cs

```diff
@@ -242,6 +242,14 @@ public interface IPublicAPI
         /// </summary>
         public void OpenUrl(Uri url, bool? inPrivate = null);
 
+        /// <summary>
+        /// Opens the URL with the given Uri object. 
+        /// The browser and mode used is based on what's configured in Flow's default browser settings.
+        /// Non-C# plugins can use this method.
+        /// </summary>
+        /// <param name="profileArg">Optional Profile name</param>
+        public void OpenUrl(string url, bool hasProfile, string profileArg);
```

Comment:
that was my first approach but turns out that if I do that, we face ambiguous calls because OpenURL with string and string(default) matches the other OpenURL with only one string.
I didn't want to refactor this to use proper objects because it would go beyond the scope of this PR as well.